src/test/java/org/broadinstitute/consent/http/db/DAOTestHelper.java

```diff
@@ -31,17 +31,19 @@
 import org.jdbi.v3.sqlobject.SqlObjectPlugin;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestPlan;
 import org.testcontainers.containers.PostgreSQLContainer;
 import org.testcontainers.containers.wait.strategy.Wait;
 
-public class DAOTestHelper extends AbstractTestHelper {
+public class DAOTestHelper extends AbstractTestHelper implements TestExecutionListener {
```

Comment:
TestExecutionListener is the magic in Junit5 that lets us build objects that can live across our tests.